StRoot/StBFChain/BigFullChain.h

```diff
@@ -1211,6 +1211,7 @@ Bfc_st BFC[] = { // standard chains
   {"McEvOut"     ,""  ,"","StMcEvent,Tree"                       ,"","","Write StMcEvent to StTree",kFALSE},
   {"EvOut"       ,""  ,"","Tree"                                   ,"","","Write StEvent to StTree",kFALSE},
   {"GeantOut"    ,""  ,"","Tree"                                ,"","","Write g2t tables to StTree",kFALSE},
+  {"Geant4Out"   ,""  ,"","Tree"                        ,"","","Write g2t tables to StTree from G4",kFALSE},
```

Comment:
Note: this chain option requires an update to StBFChain.cxx to add the StGeant4Maker into the tree, which must
wait for the integration of StGeant4Maker.